style.css

```diff
@@ -264,7 +336,10 @@ h3 {
   margin-left: 5px;
 }
 
-.see-project {
+.see-project1,
+.see-project2,
+.see-project3,
+.see-project4 {
```

Comment:
- [ ] Please add `button actions` as shown in the Figma design to your `see protect, get my resume and get in touch` buttons
- 
<img width="333" alt="bnt" src="https://user-images.githubusercontent.com/79058364/155740216-84c92582-4f30-48bb-8712-267949f87efa.png">

Reply:
resolved 👍